src/migrateStaticObservableMethodsRule.ts

```diff
@@ -0,0 +1,182 @@
+        continue;
+      }
+      const importStatementStart = importStatement.getStart(sourceFile);
+      const importStatementEnd = importStatement.getEnd();
+      ctx.addFailure(
+        importStatementStart,
+        importStatementEnd,
+        Rule.IMPORT_FAILURE_STRING,
+        Lint.Replacement.deleteFromTo(importStatementStart, importStatementEnd)
+      );
+    }
+  }
+}
+
+function isRxjsStaticOperator(node: ts.PropertyAccessExpression) {
+  return 'Observable' === node.expression.getText() && RXJS_OPERATORS.has(node.name.getText());
+}
+
+function isRxjsStaticOperatorCallExpression(node: ts.Node, typeChecker: ts.TypeChecker) {
+  // Expression is of the form fn()
+  if (!tsutils.isCallExpression(node)) {
+    return false;
+  }
+  // Expression is of the form foo.fn
+  if (!tsutils.isPropertyAccessExpression(node.expression)) {
+    return false;
+  }
+  // fn is one of RxJs instance operators
+  if (!isRxjsStaticOperator(node.expression)) {
+    return false;
+  }
+  // fn(): k. Checks if k is an observable. Required to distinguish between
+  // array operators with same name as RxJs operators.
+  if (!returnsObservable(node, typeChecker)) {
+    return false;
+  }
+  return true;
+}
+
+function findImportedRxjsOperators(sourceFile: ts.SourceFile): Set<string> {
+  return new Set<string>(
+    sourceFile.statements.filter(tsutils.isImportDeclaration).reduce((current, decl) => {
+      if (!decl.importClause) {
+        return current;
+      }
+      if (!decl.moduleSpecifier.getText().startsWith(`'rxjs'`)) {
+        return current;
+      }
+      if (!decl.importClause.namedBindings) {
+        return current;
+      }
+      const bindings = decl.importClause.namedBindings;
+      if (ts.isNamedImports(bindings)) {
+        return [
+          ...current,
+          ...(Array.from(bindings.elements) || []).map(element => {
+            return element.name.getText();
+          })
+        ];
+      }
+      return current;
+    }, [])
+  );
+}
+
+function operatorAlias(operator: string) {
+  return 'observable' + operator[0].toUpperCase() + operator.substring(1, operator.length);
+}
+
+function createImportReplacements(operatorsToAdd: Set<string>, startIndex: number): Lint.Replacement[] {
+  return [...Array.from(operatorsToAdd.values())].map(operator =>
+    Lint.Replacement.appendText(startIndex, `\nimport {${operator} as ${operatorAlias(operator)}} from 'rxjs';\n`)
+  );
+}
+
+/*
+ * https://github.com/ReactiveX/rxjs/tree/master/compat/add/observable
+ */
+const RXJS_OPERATORS = new Set([
+  'bindCallback',
+  'bindNodeCallback',
+  'combineLatest',
+  'concat',
+  'defer',
+  'empty',
+  'forkJoin',
+  'from',
+  'fromEvent',
+  'fromEventPattern',
+  'fromPromise',
+  'generate',
+  'if',
+  'interval',
+  'merge',
+  'never',
+  'of',
+  'onErrorResumeNext',
+  'pairs',
+  'rase',
+  'range',
+  'throw',
+  'timer',
+  'using',
+  'zip'
+]);
+
+// Not handling NEVER
```

Comment:
and [EMPTY](https://github.com/ReactiveX/rxjs/blob/6.0.0-rc.1/src/internal/observable/empty.ts#L8)